### DISCUSSION TITLE
New strategy for changing tables/records

### DISCUSSION THREAD

**Discussion by tigran90** [dr5hn/countries-states-cities-database#838] [Ideas]
Good afternoon, colleagues.
if we decide that my migration(https://github.com/dr5hn/countries-states-cities-database/pull/808) solution is ok, then we need to rework the strategy for changing adding and deleting records.
All changes should be in /bin/db/migrations/XXX_name.php
I don't see the point in doing it manually in SQL or somewhere else. 
In Git should have a dump database and fixes in the form of migration. 
**new regulations are needed**

**Comment by dr5hn** [comment]
@tigran90 Thanks for your patience! It needed thorough testing; hence, I was waiting to find some time to sit down with it. Finally, your changes are live and in use.

I've removed all unnecessary scripts and henceforth we will be using the migration folder to track the db updates.

Thanks so much for making this happen!
